apps/style/common.scss

```diff
@@ -1201,6 +1201,22 @@ $workspace-header-button-margin: ($workspace-headers-height - $workspace-header-
   }
 }
 
+.show-code {
+  summary {
+    font-size: 18px;
+    line-height: 20px;
+    color: #5b6770;
+    font-family: "Gotham 5r", sans-serif;
+    font-weight: normal;
+    outline: none;
+    padding: 5px;
+  }
+
+  .generated-code-container {
+    margin: 25px 10px 25px -80px;
+  }
+}
+
```

Comment:
For stuff like colors, I've started moving some of them into sharedJsxStyles.js. Font could probably go in here as well, tho it does mean we have one copy in css and another in js.